tests/factories/models.py

```diff
@@ -93,7 +93,7 @@ class Meta:
     last_login = factory.Faker("date_object")
     location = factory.Faker("city")
     profile_url = factory.LazyAttribute(
-        lambda obj: f"https://www.example.com/{obj.display_name}"
+        lambda obj: f"https://ex.com/{obj.display_name}",
```

Comment:
This was generating URLS that were greater than our allowed 40 characters configured on the column